whylog_vim/gui/files_manager.py

```diff
@@ -0,0 +1,26 @@
+import vim
+from whylog_vim.gui.vim_commander import VimCommander
+
+
+class FilesManager(object):
+
+    @staticmethod
+    def get_files_window_id(file_name):
+        for window_id, window in enumerate(vim.windows, 1):
+            if window.buffer.name.endswith(file_name):
+                return window_id
```

Comment:
The same question again:
https://github.com/9livesdata/whylog-vim/pull/2#discussion_r61730847